deps/npm/LICENSE

```diff
@@ -1,11 +1,29 @@
+The npm application
 Copyright (c) npm, Inc. and Contributors
-All rights reserved.
+Licensed on the terms of The Artistic License 2.0
 
-npm is released under the Artistic License 2.0, subject to additional terms
-that are listed below.
```

Comment:
@Fishrock123 
This **was** an issue since Artistic doesn't allow additional modifications to the terms.